Fix a wrong comment in qos_sai_base.py

congh-nvidia opened the pull request:
<!--
Please make sure you've read and understood our contributing guidelines;
https://github.com/sonic-net/SONiC/blob/gh-pages/CONTRIBUTING.md

Please provide following information to help code review process a bit easier:
-->
### Description of PR
<!--
- Please include a summary of the change and which issue is fixed.
- Please also include relevant motivation and context. Where should reviewer start? background context?
- List any dependencies that are required for this change.
-->

Summary:
There was a wrongly commented out line(line 15) which was introduced by a typo in the PR https://github.com/sonic-net/sonic-mgmt/pull/6869.
The test was not broken because the fixtures are also imported in the pytest plugin, but still, we need to explicitly import them in the test.
```
#from tests.common.dualtor.dual_tor_utils import upper_tor_host, lower_tor_host, dualtor_ports # noqa F401
```
### Type of change

<!--
- Fill x for your type of change.
- e.g.
- [x] Bug fix
-->

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)


### Back port request
- [ ] 201911
- [ ] 202012
- [x] 202205

### Approach
#### What is the motivation for this PR?
Fix a wrong comment introduced by PR https://github.com/sonic-net/sonic-mgmt/pull/6869.

#### How did you do it?
Remove the comment and fix the the pre-commit failures.

#### How did you verify/test it?
By automation.

#### Any platform specific information?

#### Supported testbed topology if it's a new test case?

### Documentation
<!--
(If it's a new feature, new test case)
Did you update documentation/Wiki relevant to your implementation?
Link to the wiki page?
-->

liat-grozovik commented on tests/qos/qos_sai_base.py:
```diff
@@ -242,7 +243,7 @@ def __getBufferProfile(self, request, dut_asic, os_version, table, port, priorit
         else:
             bufferProfileName = bufkeystr + dut_asic.run_redis_cmd(
-                argv = ["redis-cli", "-n", db, "HGET", keystr, "profile"])[0]
+                argv=["redis-cli", "-n", db, "HGET", keystr, "profile"])[0]
```
why no space here while on all other places we do have ' = '?

congh-nvidia replied:
Hi Liat, this is the default value defination of the function argument, no space is required here. Others are the variable assignments, spaces are required.

wangxin commented:
Hi @congh-nvidia, can you do a rebase to resolve the merge conflicts?

congh-nvidia commented:
> Hi @congh-nvidia, can you do a rebase to resolve the merge conflicts?

Hi @wangxin, I have rebased and fixed the conflicts, please review. Thanks.
